Python/CMakeLists.txt

```diff
@@ -114,6 +116,8 @@ set(PRP_AUDIO_SOURCES
 )
 set(PRP_AUDIO_HEADERS
     PRP/Audio/pyAudible.h
+    PRP/Audio/pyEAXEffects.h
+    PRP/Audio/pyEAXListenerMod.h
```

Comment:
The file is misnamed as `pyEaxListenerMod.h`, which will error on case sensitive file systems, such as Linux and macOS.